index.js

```diff
@@ -0,0 +1,142 @@
+import {
```

Comment:
**_Well done dev, Great work using module/factory.... 👍_** 

**_We need to use just 3 modules/objects.
Now, let follow the instructions from the Odin;_**

![image](https://user-images.githubusercontent.com/31897434/92641465-aeee5700-f2d6-11ea-859f-3775b22a0802.png)
https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript



**_In another word; You need to create 3 separate objects, don't forget, no global variable._**
1. **Gameboard**: uses an array to store the 'moves'
2. **Player**: players will be store as an object
3. **Game**: control the overflow of the game.